pkg/costmodel/cluster.go

```diff
@@ -302,11 +308,17 @@ func ClusterDisks(client prometheus.Client, provider models.Provider, start, end
 			diskMap[key] = &Disk{
 				Cluster:   cluster,
 				Name:      name,
-				Breakdown: &ClusterCostsBreakdown{},
+				Breakdown: nil,
 				Local:     true,
 			}
+			if env.IsAssetModeBreakdownEnabled() {
+				diskMap[key].Breakdown = &ClusterCostsBreakdown{}
+			}
+		}
+
+		if diskMap[key].Breakdown != nil {
```

Comment:
clarifying question: Should we check if key is found in the map here Niko?

Reply:
Right above, we ensure it will exist. (Line 307, if it does not exist, then on 308 we create it.) I like your thoroughness, though, and am open to an extra safety check here.

Reply:
I see in line 307... You good! thanks!